contents/handbook/growth/yc-onboarding.md

```diff
@@ -0,0 +1,81 @@
+---
+title: YC Onboarding
+sidebar: Handbook
+showTitle: true
+---
+
+As a YCombinator company ourselves, we want to be supportive of other YC companies that want to use PostHog. As such, our onboarding process for YC companies is different and more extensive than our general process.
+
+## The YC Deal
+
+Our YC Deal is as follows:
+
+One year of Enterprise Cloud with up to 10 million events per month *or* a one-year enterprise license if they want to self-host PostHog.
+
+Companies also get a 50$ gift card to spend on our merch and enterprise support (i.e. a private Slack with our core team members).
+
+For YC W21 companies, we also offer:
+
+- A choice of Airpods or Allbirds shoes
+- A referral scheme to get more of the above
+
+> **Note:** Batch-specific deals will be updated here as new batches come along.
+
+## Onboarding Call
+
+### Inbound Request Sources
+
+YC companies can reach out to us in many ways. Ideally we will steer them towards the [dedicated YC Onboarding page](/yc-onboarding), but they may also reach out to us on _yc@posthog.com_ or _ycdeal@posthog.com_ (as well as individual emails and _hey@posthog.com_).
+
+Having received a demo request through one of these sources, schedule an appropriate 1h slot and make sure at least one person is certainly able to attend it.
+
+### Before the call
+
+Ideally we want users to have gotten started with PostHog before the demo call. From our YC Onboarding page:
+
+<div class="blog-quote">
+Before the call, we recommend you sign up to our Cloud version and try your hand at installing our snippet on your website. Doing so would allow us to tailor the onboarding to your needs, leveraging data from your own website instead of demo data. However, if you prefer to get a demo first, we'll give you access to a demo playground during our call.
```

Comment:
Any reason we're pushing Cloud here?

Reply:
Simply easier to get set up with for the call - they can always ditch the account